connector/src/test/java/com/google/cloud/spark/bigquery/v2/BigQueryInputPartitionReaderTest.java

```diff
@@ -81,7 +81,7 @@ public void testReadAvro() throws Exception {
                         ALL_TYPES_TABLE_AVRO_RAW_SCHEMA
                 );
 
-        BigQueryInputPartitionReader reader = new BigQueryInputPartitionReader(readRowsResponses, converter);
+        BigQueryInputPartitionReader reader = new BigQueryInputPartitionReader(readRowsResponses, converter, null);
```

Comment:
why is this null? Won't it give null pointer exception?

Reply:
No, as I'm not calling `close()` in the test. The input is mocked anyway